.github/CODEOWNERS

```diff
@@ -1049,6 +1049,11 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 
 
 # Data Discovery
+/x-pack/test/common/services/ingest_manager.ts @elastic/kibana-data-discovery
+/x-pack/test/api_integration/services/data_view_api.ts @elastic/kibana-data-discovery
+/test/functional/fixtures/kbn_archiver/dashboard_error_cases.json @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/makelogs @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/large_fields @elastic/kibana-data-discovery # Assigned per only use: test/functional/apps/management/data_views/_test_huge_fields.ts
 /x-pack/test/functional/fixtures/kbn_archiver/kibana_scripted_fields_on_logstash.json @elastic/kibana-data-discovery # Assigned per only use: https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/discover/async_scripted_fields.ts#L35
 /x-pack/test/functional/es_archives/getting_started/shakespeare @elastic/kibana-data-discovery
```

Comment:
Looks like it's from Presentation team

Reply:
I presume you mean all 4 of the highlighted lines.

Reply:
https://github.com/elastic/kibana/pull/201648/commits/a1b738959cf9e8ef8459ea895e66b5cbbe23c9ff